0_From_Code_First_To_First_Principles/1_code_first/assignment/src/user/userController.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response, NextFunction } from "express";
+import { UserService } from "./userService";
+
+export class UserController {
+	static async createUser(req: Request, res: Response, next: NextFunction) {
+		const userService = new UserService();
```

Comment:
curious as to why you decided to instantiate a new `UserService` instance on each `UserController` method invocation?

Reply:
I guess I didn't think it though :) good catch!